Artistic Style Transfer with CycleGAN/Readme.md

```diff
@@ -0,0 +1,14 @@
+Artistic Style Transfer With CycleGAN :
+
+* In this [notebook](https://colab.research.google.com/drive/19t6m05nv7gnVdRZMkD-XZnaWUIsKcKZF?usp=sharing) we will explore the Artistry of the GANs , that is we will create Van Gogh Style pictures , by learning from his paintings and implement his ***Artistic*** style on Normal images. 
```

Comment:
Not Required.

Just put a section in the readme as Style Transfer and have three columns: Content, Style and Output

All images should be in assets folder